source/mainnet/smart-contracts/guides/build-schema.rst

```diff
@@ -185,32 +186,93 @@ If using ``cargo concordium`` version 2.6.0 or newer then the schema can be
 output in JSON format that can be more suitable for use in dApps. When building
 the contract use ``--schema-json-out DIR`` to output the schema for each
 contract in the module to a JSON file inside the directory ``DIR``. The
-directory must exist.
+directory must exist. The files created will be named after the smart contract
+names that exist in the module or a counter in case of special characters in the smart contract names:
 
 .. code-block:: console
 
-   $cargo concordium build --schema-json-out "/some/path"
+   $cargo concordium build --schema-json-out "/some/path/"
 
-Both ``--schema-out`` and ``--schema-json-out`` can be used at the same time and
-schemas in both formats will be output.
+If using ``cargo concordium`` version 2.7.0 or newer then the schema can be
+output in base64 format that is currently supported in the browser wallet. When building
```

Comment:
Can you include the variables.rst at the top of the file and then replace browser wallet with |bw| so that it gets the correct product name?